content/posts/2020-06-14-how-does-the-pipe-operator-actually-work.Rmarkdown

```diff
@@ -93,7 +93,7 @@ Why is this useful? Because you can actually modify this quoted expression which
 mtcars %>% subset(cyl == 4, select = c(hp, cyl))
 ```
 
-This looks interesting. Apparently you can subset a quoted expression like a list using `[[`. When you have a quoted expression `expr` that involves a function, `expr[[1L]]` will get you the name of the function and `expr[-1L]` will get you the arguments of the function.
+This looks interesting. Apparently you can subset a quoted expression like a list using `[[`. When you have a quoted expression `expr` that is a function call, `expr[[1L]]` will get you the name of the function and `expr[-1L]` will get you the parameters supplied to the arguments of the function.
```

Comment:
clarify the subject as function call, avoiding being mistaken for function definition.